foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/platform/database/OraclePlatform.java

```diff
@@ -951,14 +951,16 @@ public boolean useJDBCStoredProcedureSyntax() {
     protected String MAX_ROW = "WHERE ROWNUM <= ";
     protected String MIN_ROW = ") WHERE rnum > ";
     // Bug #453208
-    protected String LOCK_START_PREFIX = " AND (";
+    protected String LOCK_START_PREFIX_AND = " AND (";
```

Comment:
is this rename required? Keep in mind that during the backport, one will have to keep the original name (compatibility)